### PR TITLE
Base policy

### PR DESCRIPTION
**Pull request by aj3sh**
This is a base organization policy. Further policies can discussed and updated later.

### CODE REVIEW

File: policy/README.md [hunk 1]
```diff
@@ -1 +1,50 @@
 # OpenSource Nepal Policy
+
+---
+
+#### **Repository Management**
+
+- **New Repository Creation**:
+  - All new repository ideas must be discussed in the organization's Discord channel.
+  - Any member can create a public repository. Admins must review the repository later and decides whether it should remain or be archived/deleted.
```
Comment (sugat009) [inline policy/README.md]:
```suggestion
  - Any member can create a public repository. Admins must review the repository later and decide whether it should remain or be archived/deleted.
```

File: policy/README.md [hunk 2]
```diff
@@ -1 +1,50 @@
...
+- **Discussions**: All members can create discussions in any repository to propose ideas, report issues, or provide feedback.
+
+---
+
+#### **Pull Requests**
+
+- **Minimum number of approvals for merging a PR**:
+  - **Contributors ≥ 10**: 2 approvals
+  - **Contributors < 10 but > 1**: 1 approval
+  - **1 Contributor**: No approval needed
```
Comment (sugat009) [inline policy/README.md]:
```suggestion
  - **Contributors < 10**: 1 approval
```
I think every contribution and every PR should have at least 1 approval.

**Review by sugat009 — CHANGES_REQUESTED** [review]

File: policy/README.md (hunk 2 above)
Comment (aj3sh) [reply policy/README.md]: Who approves when there is only one maintainer on a repo? 

This is for the initial phase, when there is single maintainer, and does the initial project work.

Comment (sugat009) [reply policy/README.md]: Organization admins should approve in that case or there is no meaning in sending a PR. A direct push to the `main` branch should work instead. When the repo is setup then the maintainer can change the rules to not allow direct push to the `main` branch.

Comment (aj3sh) [reply policy/README.md]: Organization admins aren't responsible for approving every PRs. But direct push to the main branch should be good.

**Review by sugat009 — APPROVED** [review]

**Pull request merged**